src/Nowruz/modules/settings/components/orgTeam/useOrgTeam.tsx

```diff
@@ -0,0 +1,161 @@
+
+export const useOrgTeam = () => {
+  const currentIdentity = useSelector<RootState, CurrentIdentity | undefined>((state) =>
+    state.identity.entities.find((identity) => identity.current),
+  );
+
+  const [addedMembers, setAddedMembers] = useState<string[]>([]);
+  const [teamMembers, setTeamMembers] = useState<AccountItem[]>([]);
+  const [selectedPerson, setselectedPerson] = useState();
+  const [openAlert, setOpenAlert] = useState(false);
+  const [toDeleteName, setToDeleteName] = useState('');
+  const [toDeleteId, settoDeleteId] = useState('');
+  const [page, setPage] = useState(1);
+  const [hasMore, setHasMore] = useState(true);
+
+  const getTeamMembers = async () => {
+    try {
+      const res = await getOrganizationMembers(currentIdentity!.id, { page: 1 });
+      const mapped = res.items.map((item) => {
+        return {
+          id: item.id,
+          img: item.avatar?.url,
+          type: 'users',
+          name: `${item.first_name || ''} ${item.last_name || ''}`,
+          username: item.username,
+        };
+      });
+      setTeamMembers(mapped);
+      setPage(1);
+      setHasMore(true);
+    } catch (e) {
+      console.log('error in getting team members', e);
+    }
+  };
+
+  const loadMore = async () => {
+    try {
+      if (hasMore) {
+        const res = await getOrganizationMembers(currentIdentity!.id, { page: page + 1 });
+        if (res.items.length) {
+          const mapped = res.items.map((item) => {
+            return {
+              id: item.id,
+              img: item.avatar?.url,
+              type: 'users',
+              name: `${item.first_name || ''} ${item.last_name || ''}`,
+              username: item.username,
+            };
+          });
+          setTeamMembers([...teamMembers, ...mapped]);
+          setPage(page + 1);
+        } else setHasMore(false);
+      }
+    } catch (e) {
+      console.log('error in getting team members', e);
+    }
+  };
+
+  useEffect(() => {
+    getTeamMembers();
+  }, []);
+
+  const followingToOption = (followings: Following[]) => {
+    return followings.map((i) => ({
+      label: JSON.stringify({ label: i.identity_meta.name || '', description: i.identity_meta.username }),
+      value: i.identity_meta.id,
+      icon: <Avatar img={i.identity_meta.avatar} type="users" size="24px" />,
+    }));
+  };
+
+  const searchMembers = async (searchText: string, cb) => {
+    try {
+      if (searchText) {
+        const response = await filterFollowings({ name: searchText, type: 'users', page: 1, limit: 10 });
+        const filtered = response.items.filter((item) => !teamMembers.map((m) => m.id).includes(item.identity_meta.id));
+        cb(followingToOption(filtered));
+      }
+    } catch (error) {
+      console.error('Error fetching following data:', error);
+    }
+  };
+
+  const onSelectMember = (member) => {
+    setselectedPerson(member);
+    const members = [...addedMembers];
+    const indx = members.findIndex((item) => item === '');
```

Comment:
`members.filter(member => !!member)` or `const members = addedMembers.filter(Boolean);`
instead of findIndex and splice